package.json

```diff
@@ -230,12 +230,12 @@
 		"postinstall": "node ./node_modules/vscode/bin/install"
 	},
 	"dependencies": {
-		"vscode-debugadapter": "^1.11.0",
-		"vscode-debugprotocol": "^1.11.0",
+		"vscode-debugadapter": "^1.12.0",
```

Comment:
If this all still works on Stable with no issues; sounds safe to me. If we build anything on any new APIs we might need to be careful, but I guess whatever new features we add we'll test on Stable whether we think they use new APIs or not.

I'm not too worried about people on pre-stable versions; if people update Code as frequently as Dart Code it seems unlikely they'll be stuck around on old versions.